packages/mco/components/modals/drpolicy-apps-apply/apply-policy-modal.tsx

```diff
@@ -267,7 +269,9 @@ const ApplyDRPolicyModal: React.FC<CommonModalProps<ApplyModalExtraProps>> = (
                 variant={state.message.type}
                 title={state.message.text}
                 className="odf-alert odf-alert--margin-top"
-              />
+              >
+                {state.message.description}
```

Comment:
```suggestion
                {state.message?.description}
```
description is optional, it will break